src/dpl/README.md

```diff
@@ -48,10 +64,100 @@ is supported, so `FILL*` will match, e.g., `FILLCELL_X1 FILLCELL_X16 FILLCELL_X2
 FILLCELL_X32 FILLCELL_X4 FILLCELL_X8`.  To specify a different naming prefix
 from `FILLER_` use `-prefix <new prefix>`.
 
+```tcl 
+filler_placement
+    [-prefix prefix]
+    filler_masters
+```
+
+#### Options
+
+| Switch Name | Description |
+| ----- | ----- |
+| `-prefix` | Prefix to name the filler cells (default "FILLER_"). |
+| `filler_masters` | Filler master cells. | 
+
+### Remove Fillers
+
+```tcl
+remove_fillers 
+```
+
+No arguments are needed for this function. 
+
+### Check Placement
+
+The `check_placement` command checks the placement legality. It returns
+`0` if the placement is legal.
+
+```tcl
+check_placement
+    [-verbose]
+    [-disallow_one_site_gaps]
+```
+
+#### Options
+
+| Switch Name | Description |
+| ----- | ----- |
+| `-verbose` | Enable verbose logging. |
+| `-disallow_one_site_gaps` | Disable one site gaps during placement. |
```

Comment:
```suggestion
| `-disallow_one_site_gaps` | Disable one site gap during placement check. |
```

Reply:
A note about using the same as used in `detailed_placement` to guarantee consistency would be nice.